modules/tinymce/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Fixed
+- Resizing table columns in some scenarios would resize the column to an incorrect position #TINY-7731
```

Comment:
Do we need to specify that this is specifically while using the table plugin? Is this specifically while using the table plugin?

Reply:
You can only resize table columns when using the table plugin, so I don't see it as necessary.